src/Parser/DisciplinePriest/CombatLogParser.js

```diff
@@ -170,6 +182,17 @@ class CombatLogParser extends MainCombatLogParser {
           </dfn>
         )}
       />,
+      this.modules.atonement.active && (
+        <StatisticBox 
+        icon={<SpellIcon id={SPELLS.ATONEMENT1.id} />}
+        value= {this.modules.atonement.improperAtonementRefreshes.length}
+        label={(
+          <dfn data-tip={`${((this.modules.atonement.improperAtonementRefreshes.length / this.modules.atonement.totalAtonementRefreshes * 100) || 0).toFixed(2)}% (${this.modules.atonement.improperAtonementRefreshes.length}/${this.modules.atonement.totalAtonementRefreshes}) of your Atonement refreshes were too early. ${((this.modules.atonement.improperAtonementRefreshes.length / this.modules.atonement.totalAtones * 100) || 0).toFixed(2)}% (${this.modules.atonement.improperAtonementRefreshes.length}/${this.modules.atonement.totalAtones}) of all Atonements applied were refreshed too early.` }>
```

Comment:
That makes sense, I was actually having trouble structuring the tooltip to be something that is important. The suggested tooltip makes more sense I will update.